lib/vsc/mympirun/mpi/openmpi.py

```diff
@@ -67,7 +95,7 @@ def pinning_override(self):
         self.log.debug("pinning_override: type %s ", override_type)
 
         ranktxt = ""
-        sockets_per_node = 2
+        sockets_per_node = self.determine_sockets_per_node()
```

Comment:
In order to do the pinning as requested, `mympirun` needs to know the number of sockets in the node (currently 2 in all our systems, but it shouldn't be hardcoded)

This is only required when using OpenMPI and when `--overridepin` is used, see https://github.com/hpcugent/vsc-mympirun/blob/develop/lib/vsc/mympirun/mpi/mpi.py#L324 and https://github.com/hpcugent/vsc-mympirun/blob/develop/lib/vsc/mympirun/mpi/mpi.py#L1034